tests/integration/defs/perf/nvdf.py

```
@@ -0,0 +1,578 @@
+    os.getenv("HOST_NODE_NAME", "")
+    build_id = os.getenv("BUILD_ID", "")
+    build_url = os.getenv("BUILD_URL", "")
+    job_name = os.getenv("JOB_NAME", "")
+
+    # Get gpu_type and gpu_count from nvidia-smi
+    gpu_type = ""
+    gpu_count = 0
+    try:
+        # Get GPU names from nvidia-smi
+        result = subprocess.run(
+            ["nvidia-smi", "--query-gpu=name", "--format=csv,noheader"],
```

Ruff (0.14.1)

Starting a process with a partial executable path

(S607)